resources/lang/en/header.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Pagination Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the paginator library to build
+    | the simple pagination links. You are free to change them to anything
+    | you want to customize your views to better match your application.
+    |
+    */
+
+    'ePromotions' => 'ePromotions',
```

Comment:
[rank C] the key name should be follow underscore style.
and 'ePromotions' key should be 'site_name' or 'system_name'
'eP' key should be 'stand_of_sys_name'